tests/manage/pv_services/test_pvc_evict_ceph_clients.py

```diff
@@ -0,0 +1,119 @@
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["node"],
+        argvalues=[
+            pytest.param(
+                "same",
+                marks=[
+                    pytest.mark.bugzilla("1901499"),
+                    pytest.mark.polarion_id("OCS-3985"),
+                ],
+            ),
+            pytest.param(
+                "different",
+                marks=[pytest.mark.polarion_id("OCS-3984")],
+            ),
+        ],
+    )
+    def test_pvc_evict_ceph_clients(
+        self, node, pvc_factory, teardown_factory, pod_factory
+    ):
+        worker_nodes = get_worker_nodes()
+
+        # create a RWX PVC
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            status=constants.STATUS_BOUND,
+        )
+
+        # create a pod in particular node
+        selected_node = random.choice(worker_nodes)
+        logger.info(f"creating first pod on node {selected_node}")
+        first_pod_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj,
+            node_name=selected_node,
+            pod_dict_path=constants.NGINX_POD_YAML,
+        )
+        helpers.wait_for_resource_state(
+            resource=first_pod_obj, state=constants.STATUS_RUNNING, timeout=120
+        )
+
+        # fetch the sub-volume path
+        pvc_obj.reload()
+        sub_volume_path = OCP(kind="pv").get(resource_name=pvc_obj.backed_pv)["spec"][
+            "csi"
+        ]["volumeAttributes"]["subvolumePath"]
+
+        # evict ceph-fs clients
+        logger.info("Evicting ceph-fs clients!")
+        ceph_tools_pod_obj = get_ceph_tools_pod()
+        cmd_ls = "ceph tell mds.0 client ls"
+        output_ls_cmd = json.loads(
+            ceph_tools_pod_obj.exec_sh_cmd_on_pod(command=cmd_ls)
+        )
+        ids = list()
+        for instance in output_ls_cmd:
+            if instance["client_metadata"]["root"] == sub_volume_path:
+                ids.append(instance["id"])
+        logger.info(f"Client Ids to evict: {ids}")
+        try:
+            for id in ids:
+                cmd_evict = f"ceph tell mds.0 client evict id={id}"
+                ceph_tools_pod_obj.exec_sh_cmd_on_pod(command=cmd_evict)
+        except CommandFailed as e:
+            raise ValueError(f"[Error] Client eviction failed: {e}")
+        else:
+            logger.info("Clients are evicted successfully!")
+
+        # running second pod on same/different node
+        if node == "different":
+            worker_nodes.remove(selected_node)
+            selected_node = random.choice(worker_nodes)
+
+        logger.info(f"Running second pod on {selected_node} node")
+        second_pod_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj,
+            node_name=selected_node,
+            pod_dict_path=constants.NGINX_POD_YAML,
+        )
+        helpers.wait_for_resource_state(
```

Comment:
Same as above you can remove it and add to pod_factory `status=constants.STATUS_RUNNING,`